src/components/ByState.tsx

```diff
@@ -11,17 +12,22 @@ function ByState(props) {
 
     const [selectedState, setSelectedState] = useState(null)
     const [stateData, setStateData] = useState(null)
-    const [total, setTotal] = useState(null)
+    const [total, setTotal] = useState<number|null>(null)
+
+    const mapHandler = (e: React.PointerEvent<SVGElement>) => {
```

Comment:
The event handler type was a bit of a mystery to me. A Stack Overflow post recommended to use the PointerEvent type. I had to do this so I could use e.target.dataset a couple lines down.